firebase.json

```diff
@@ -48,6 +48,56 @@
           "destination": "https://zksync.io/contact",
           "type": "302"
         },
+        {
+          "source": "/docs/dev/building-on-zksync/hello-world.html",
+          "destination": "/build/quick-start/hello-world.html",
+          "type": "302"
+        },
+        {
+          "source": "/docs/reference/architecture/differences-with-ethereum.html",
+          "destination": "/build/developer-reference/differences-with-ethereum.html",
+          "type": "302"
+        },
+        {
+          "source": "/docs/reference/concepts/hyperscaling.html",
+          "destination": "/zk-stack/concepts/hyperchains-hyperscaling.html",
+          "type": "302"
+        },
+        {
+          "source": "/docs/tools/",
+          "destination": "/build",
+          "type": "302"
+        },
+        {
+          "source": "/docs/tools/zksync-cli",
+          "destination": "/build/tooling/zksync-cli/getting-started.html",
+          "type": "302"
+        },
+        {
+          "source": "/docs/tools/hardhat",
+          "destination": "/build/tooling/hardhat/getting-started.html",
+          "type": "302"
+        },
+        {
+          "source": "/docs/dev/tutorials/cross-chain-tutorial.html",
+          "destination": "/build/tutorials/smart-contract-development/cross-chain-tutorial.html",
+          "type": "302"
+        },
+        {
+          "source": "/docs/dev/tutorials/custom-aa-tutorial.html",
```

Comment:
this is duplicate with the main branch (there are already two for prod and staging)